tests/network/bindingplugin_macvtap.go

```diff
@@ -34,7 +34,8 @@ import (
 	v1 "kubevirt.io/api/core/v1"
 
 	"kubevirt.io/kubevirt/pkg/libvmi"
-	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
+	"kubevirt.io/kubevirt/pkg/virt-config/featuregate"
+
```

Comment:
nit: remove new line

Reply:
The newline separates production from e2e tests packages.